protocol/0088-PRBD-price_builder.md

```diff
@@ -0,0 +1,13 @@
+# Price Builder
+
+## Summary
+
+This specification adds a flexible way to configure "price" method for different purpose.
+
+For perpetual futures markets there should be a “mark price” configuration and a “market price for funding” configuration so that the market can, potentially use different mark price for mark-to-market and price monitoring and completely different price for calculating funding.
+
+## Acceptance criteria
```

Comment:
We should also move all the acceptance criteria that are about how the "price is built" and the parameter validation here as that's common across both mark price and the "internal twap price".